$NetBSD: patch-configure,v 1.1 2015/09/26 23:29:13 gdt Exp $

Use of objcopy may be patched away, discussed with upstream.

--- configure.orig	2015-02-26 16:52:11.000000000 +0000
+++ configure
@@ -3184,10 +3184,6 @@ $as_echo "no" >&6; }
 fi
 
 
-if test -z "$OBJCOPY"
-then
-  as_fn_error $? "objcopy not found. Make sure you have GNU binutils installed." "$LINENO" 5
-fi
 
 # check for install
 # Extract the first word of "install", so it can be a program name with args.
